CHANGES.md

```diff
@@ -53,6 +53,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * [Marathon] Introduce new exit error code when the framework was removed from Mesos.
 * Ensure Bouncer uses configured web proxy. (DCOS_OSS-5167)
+* Added Fluent Bit metrics to the pipeline (DCOS-54425)
```

Comment:
This entry should be added at the top of the section, with a following blank line. See https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines#format

Reply:
ugh, looks like a rebase artefact, thanks for catching it

Reply:
upgefixed

Reply:
Thank you @branden for helping out here! :-)

Most importantly, this should have gone into the "in development" section, for 1.13.2 :-) haha... doing that now in an editorial update.